cg/apps/demultiplex/sample_sheet/read_sample_sheet.py

```diff
@@ -43,32 +43,33 @@ def validate_samples_unique_per_lane(samples: List[FlowCellSample]) -> None:
         validate_samples_are_unique(samples=lane_samples)
 
 
-def get_sample_sheet_from_file(
-    infile: Path,
-    flow_cell_sample_type: Type[FlowCellSample],
-) -> SampleSheet:
+def get_sample_sheet_from_file(infile: Path) -> SampleSheet:
     """Parse and validate a sample sheet from file."""
     sample_sheet_content: List[List[str]] = ReadFile.get_content_from_file(
         file_format=FileFormat.CSV, file_path=infile
     )
+    sample_type: Type[FlowCellSample] = get_sample_type(infile)
+
     return get_validated_sample_sheet(
         sample_sheet_content=sample_sheet_content,
-        sample_type=flow_cell_sample_type,
+        sample_type=sample_type,
     )
 
 
-def get_sample_type_from_sequencer_type(sequencer_type: str) -> Type[FlowCellSample]:
-    bcl_converter: str = get_bcl_converter_by_sequencer(sequencer_type)
-    if bcl_converter == BclConverter.BCL2FASTQ:
-        return FlowCellSampleBcl2Fastq
-    return FlowCellSampleBCLConvert
-
-
-def get_bcl_converter_by_sequencer(sequencer_type: str) -> str:
-    """Return the BCL converter based on the sequencer."""
-    if sequencer_type in [Sequencers.NOVASEQ, Sequencers.NOVASEQX]:
-        return BclConverter.DRAGEN
-    return BclConverter.BCL2FASTQ
+def get_sample_type(sample_sheet_path: Path) -> Type[FlowCellSample]:
```

Comment:
I think this function warrants a simple doc string. Do you think it's superfluous? 
```suggestion
def get_sample_type(sample_sheet_path: Path) -> Type[FlowCellSample]:
    """Returns the sample type based on the header of the given sample sheet"""
```